### PR TITLE
E2E update for next

### PR DESCRIPTION
**Pull request by palango**
**Thank you for submitting this pull request :)**

Fixes https://github.com/raiden-network/light-client/issues/2699

**Short description**

- Update synapse, include new presence module
- Update geth to Berlin-compatible version
- Update to Raiden 2.0
- Update PFS

## TODO
- [ ] Upload new image

**Definition of Done**

- [ ] Steps to manually test the change have been documented
- [ ] Acceptance criteria are met
- [ ] Change has been manually tested by the reviewer (dApp) 

**Steps to manually test the change (dApp)**

1.
2.

### CODE REVIEW

**Comment by andrevmatos** [comment]
@palango #2750 should be compatible with the current protocol, signatures & transport from PC. Could you please test current `develop` branch of PC with that branch of the LC, and report if it works?

**Review by andrevmatos — APPROVED** [review]
LGTM!. That's quite a bit of changes to our e2e setup, but it seems compliant with current transport approach. Thank you!

File: raiden-ts/tests/e2e/e2e.spec.ts [hunk 1]
```diff
@@ -60,6 +60,9 @@ async function createRaiden(account: number | string | Signer): Promise<Raiden>
       confirmationBlocks: 5,
       autoSettle: false, // required to use `settleChannel` later
       autoUDCWithdraw: false, // required to use `withdrawFromUDC` later
+      caps: {
+        [Capabilities.RECEIVE]: 1,
+      }
```
Comment (andrevmatos) [inline raiden-ts/tests/e2e/e2e.spec.ts:65]: Linter is complaining
```suggestion
      },
```

**Comment by palango** [comment]
e2e image v1.1.3 has been uploaded to ducker hub.

**Pull request merged**